src/purchases.ts

```diff
@@ -321,15 +296,15 @@ export default class Purchases {
      * Fetch the product info
      * @param {String[]} productIdentifiers Array of product identifiers
      * @param {String} type Optional type of products to fetch, can be inapp or subs. Subs by default
-     * @returns {Promise<PurchasesProduct[]>} A promise containing an array of products. The promise will be rejected
+     * @returns {Promise<PurchasesStoreProduct[]>} A promise containing an array of products. The promise will be rejected
      * if the products are not properly configured in RevenueCat or if there is another error retrieving them.
      * Rejections return an error code, and a userInfo object with more information. The promise will also be rejected
      * if setup has not been called yet.
      */
     public static async getProducts(
         productIdentifiers: string[],
         type: PURCHASE_TYPE = PURCHASE_TYPE.SUBS
-    ): Promise<PurchasesProduct[]> {
+    ): Promise<PurchasesStoreProduct[]> {
```

Comment:
I'm not sure why we've always prefixed these, is it a common thing in Typescript?

Reply:
No idea!